packages/components/src/components/post-tab-panel/post-tab-panel.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, Element, h, Host, Prop, State } from '@stencil/core';
+import { version } from '../../../package.json';
+
+@Component({
+  tag: 'post-tab-panel',
+  styleUrl: 'post-tab-panel.scss',
+  shadow: true,
+})
+export class PostTabPanel {
+  @Element() host: HTMLPostTabPanelElement;
+
+  @State() panelId: string;
+
+  /**
+   * The name of the panel, used to associate it with a tab header.
+   */
+  @Prop() readonly name: string;
+
+  componentWillLoad() {
+    // get the id set on the host element or use a random id by default
+    this.panelId = this.host.id || crypto.randomUUID();
+  }
+
+  render() {
+    return (
+      <Host data-version={version} id={this.panelId}>
+        <div
+          aria-labelledby={`${this.panelId}--tab`}
```

Comment:
This won't work since different random id's will be generated for every component:

<img width="506" alt="image" src="https://github.com/swisspost/design-system/assets/1659006/a2a34a9f-ff06-48ed-ab0f-d681361f578d">

It's important that only one component generates the ID and distributes it to the others. We could let the post-tab-header generate the id and write it from there on the connected post-tab-panel in the componentDidLoad.

Reply:
I updated the behavior: now the header and panel both generate their ID's separately then, once all headers and panels are fully loaded the post-tabs gets the ID's and uses them to set aria attributes.